db/src/main/kotlin/edu/artic/db/MoshiAdapters.kt

```diff
@@ -63,4 +65,36 @@ class ZonedDateTimeAdapter {
 
     @FromJson
     fun fromText(text: String): ZonedDateTime = ZonedDateTime.parse(text, formatter)
-}
\ No newline at end of file
+}
+
+/**
+ * Parses floor as an integer. We default to [Int.MIN_VALUE] as 0 is a valid floor.
```

Comment:
Update text since we now have an `INVALID_FLOOR` constant. Maybe just add a blank line, then `@see INVALID_FLOOR`.

Reply:
Modified the comment.